nnpdf_data/nnpdf_data/new_commondata/STAR_2015_1JET_200GEV/filter.py

```diff
@@ -0,0 +1,154 @@
+import glob
+
+import pandas as pd
+import yaml
+
+from nnpdf_data.filter_utils.uncertainties import symmetrize_errors
+from nnpdf_data.filter_utils.correlations import compute_covmat
+
+
+def read_data(fnames):
+    df3 = pd.DataFrame()
+    df4 = pd.DataFrame()
+    for fname in fnames:
+        if "3" in fname:
+            dfi = pd.read_csv(fname)
+            if "pT" in fname:
+                correlation = False
+                df3["pT"] = dfi["Parton Jet $p_T$ [GeV/c]"]
+
+            elif "ALL" in fname:
+                df3["ALL"] = dfi["$A_{LL}$"]
+                df3["stat_min"] = dfi["stat +"]
+                df3["stat_max"] = dfi["stat -"]
+                df3["sys_min"] = dfi["sys +"]
+                df3["sys_max"] = dfi["sys -"]
+
+            df3["eta_min"] = -0.5
+            df3["eta_max"] = 0.5
+            df3["eta"] = 0.0
+            df3["sqrts"] = 200
+
+        elif "4" in fname:
+            dfi = pd.read_csv(fname)
+            if "pT" in fname:
+                correlation = False
+                df4["pT"] = dfi["Parton Jet $p_T$ [GeV/c]"]
+
+            if "ALL" in fname:
+                df4["ALL"] = dfi["$A_{LL}$"]
+                df4["stat_min"] = dfi["stat +"]
+                df4["stat_max"] = dfi["stat -"]
+                df4["sys_min"] = dfi["sys +"]
+                df4["sys_max"] = dfi["sys -"]
+
+            df4["eta_min"] = -1.0
```

Comment:
This is not really consistent with the table. As the condition is table3 -> eta < 0.5, table4 -> 0.5 < eta < 1.0, 
so we need to find a better solution...

Reply:
Maybe let's change the variable from `eta` to `abs_eta`